modules/serverless-event-transformation-deployment-patterns.adoc

```diff
@@ -0,0 +1,136 @@
+spec:
+  serviceAccountName: event-watcher
+  resources:
+    - apiVersion: v1
+      kind: Event
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: EventTransform
+      name: event-transformer
+---
+apiVersion: eventing.knative.dev/v1alpha1
+kind: EventTransform
+metadata:
+  name: event-transformer
+spec:
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: default
+  jsonata:
+    expression: |
+      # transformation expression
+----
+
+[id="serverless-event-transformation-trigger-to-service_{context}"]
+== Trigger to Service event transformation
+
+In this pattern, events flow through the sequence Broker → Trigger → EventTransform → Service or Sink. The Broker receives all events, the Trigger filters them based on defined attributes, and the filtered events are then routed to an `EventTransform` resource. The `EventTransform` reshapes or enriches the events before finally forwarding them to the target Service or Sink. 
+
+This flow makes it possible to tailor events for specific consumers without modifying the original event producer or affecting other subscribers. By applying transformations after filtering, you ensure that only relevant events are reshaped, which improves efficiency and reduces unnecessary processing.
+
+The following configuration filters events of type `original.event.type`, routes them to an `EventTransform`, and then delivers the transformed events to a Service:
+
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: transform-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: original.event.type
+  subscriber:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: EventTransform
+      name: event-transformer
+---
+apiVersion: eventing.knative.dev/v1alpha1
+kind: EventTransform
+metadata:
+  name: event-transformer
+spec:
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: destination-service
+  jsonata:
+    expression: |
+      # transformation expression
+----
+
+[id="serverless-event-transformation-broker-reply_{context}"]
+== Broker reply event transformation
+
+You can configure `EventTransform` without a sink so that it republishes transformed events back to the Broker. This pattern is useful when you want transformed events to be routed to additional Triggers or consumers.
+
+[NOTE]
+====
+When using the Broker reply feature, ensure the transformed events do not match the same Trigger that invoked the `EventTransform`. Otherwise, you risk creating an infinite event loop.
+====
+
+In the following example, events of type `original.event.type` are filtered by a Trigger, transformed by the `EventTransform`, and then republished to the Broker with a new type `transformed.event.type`. When you update the type or another attribute, the event is routed to other Triggers without being re-processed by the same Trigger.
```

Comment:
```suggestion
You can filter events of type `original.event.type` with a Trigger, transform them with `EventTransform`, and republish them to the Broker as `transformed.event.type` type. Updating the type or another attribute routes the event to different Triggers without reprocessing by the same Trigger.
```
Suggestion to avoid self-referential language and using noun identifier.